webapp/app/services/userService.js

```diff
@@ -0,0 +1,44 @@
+define('services/userService', [], function () {
+    function newAjaxCall(dataService, successCallback, errorCallback) {
```

Comment:
Good job creating a method with the shared code. Since this could also be shared across other services we may put it in a separated file, something like `requestHelper`